components/dashboard/src/components/InfoBox.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import info from '../images/info.svg';
+
+export default function InfoBox(p: { className?: string, children?: React.ReactNode }) {
+    return <div className={'flex rounded-md bg-gray-200 dark:bg-gray-800 text-gray-600 dark:text-gray-400 p-4 ' + (p.className || '')}>
+        <img className="w-4 h-4 m-1 ml-2 mr-4" src={info} />
+        {p.children}
```

Comment:
<img width="501" alt="Screenshot 2021-04-20 at 18 52 47" src="https://user-images.githubusercontent.com/599268/115434998-aca79000-a209-11eb-91c3-eb95fc5c009d.png">

```suggestion
        <span>{p.children}</span>
```